app/main.py

```diff
@@ -0,0 +1,107 @@
+from typing import Optional
+from fastapi import FastAPI,Response,status,HTTPException
+from fastapi.params import Body
+from pydantic import BaseModel
+from random import randrange
+app=FastAPI()
+# klasa Post nasledjuje BaseModel
+#postavljamo vrstu podataka koji treba da se unese
+#pravimo model
+class Post(BaseModel):
+    title: str
+    content: str
+    publisshed: bool = True #default value (opciono polje)ako korisnik ne unese podatke ovo ce biti vrednost tog polja
+    rating: Optional[int]= None #potpuna opciono polje koje prima int ali ako ga korisnik ne unese onda ce biti none(prazno)
+# kad koristimo path parametar id ce uvek biti string i zato moramo da ga promenimu int
+# moramo da obezbedimo da mora da bude integer i automacki ga konvetuje, ako se unese neki ranom id koji nije string izlazi grska
+def getone(id: int):
```

Comment:
We should discuss about file formatting, indents, spacing and other writing rules and convention.

Reply:
Formatting is better now. Please review doc I shared about the code comments